Add Experian Ltd

gazelle-24 opened the pull request:
Experian Ltd is credit monitoring service. Not documented, but they support 2FA codes via SMS and phone calls (https://i.imgur.com/K0QUZ0t.png).

gazelle-24 committed:
Add Experian Ltd

Experian Ltd is credit monitoring service. Not documented, but they support 2FA codes via SMS and phone calls (https://i.imgur.com/K0QUZ0t.png).

Carlgo11 reviewed (changes requested):
I suggest you use this image instead https://expcdn.co.uk/www/images/icon-32x32.png

gazelle-24 committed:
Update experianltd.png

gazelle-24 committed:
Add Teespring

Teespring is website for buying and selling customised apparel, phone cases, home accessories, etc (https://i.imgur.com/0Nk0qyq.png).

gazelle-24 committed:
Revert "Add Teespring"

This reverts commit f198f6ceff23c920efe52110e68f586f528d0760.
